code/modules/food_and_drinks/kitchen_machinery/microwave.dm

```diff
@@ -15,6 +15,7 @@
 	var/broken = 0 // ={0,1,2} How broken is it???
 	var/max_n_of_items = 10 // whatever fat fuck made this a global var needs to look at themselves in the mirror sometime
 	var/efficiency = 0
+	var/datum/looping_sound/microwave/beep_beep_beep
```

Comment:
name this variable something else that makes sense

Reply:
just call it soundloop, it's what literaly every piece of code does with soundloops.